apps/smart-contracts/core/test/integration/DepositTradeHelper.test.ts

```diff
@@ -344,65 +388,70 @@ describe('=> DepositTradeHelper', () => {
 
       await depositTradeHelper
         .connect(user)
-        .withdrawAndUnwrap(deployer.address, collateralBalanceBefore, collateralPermit)
+        .withdrawAndUnwrap(
+          deployer.address,
+          collateralBalanceBefore,
+          collateralPermit,
+          balancerParamsForWithdraw
+        )
 
       expect(await core.collateral.allowance(user.address, depositTradeHelper.address)).eq(
         ethers.constants.MaxUint256
       )
     })
 
-    it('withdraws WETH to DepositTradeHelper contract', async () => {
-      const expectedBaseTokenFromWithdrawal = await getBaseTokenAmountForWithdrawal(
-        core.collateral,
-        collateralBalanceBefore
-      )
-
+    it('withdraws WstETH to DepositTradeHelper contract', async () => {
```

Comment:
```suggestion
    it('withdraws wstETH to DepositTradeHelper contract', async () => {
```
similarly for elsewhere